lib/Jsrt/ChakraCore.h

```diff
@@ -70,6 +71,21 @@ typedef JsErrorCode(CHAKRA_CALLBACK * FetchImportedModuleCallBack)(_In_ JsModule
 /// <returns>
 ///     true if the operation succeeded, false otherwise.
 /// </returns>
+typedef JsErrorCode(CHAKRA_CALLBACK * FetchImportedModuleFromScriptCallBack)(_In_ DWORD_PTR dwReferencingSourceContext, _In_ JsValueRef specifier, _Outptr_result_maybenull_ JsModuleRecord* dependentModuleRecord);
```

Comment:
Updated in 2nd iteration.

---
In reply to: [114836637](https://github.com/Microsoft/ChakraCore/pull/2913#discussion_r114836637) [](ancestors = 114836637)